packages/opentelemetry-tracing/src/export/BatchSpanProcessorBase.ts

```diff
@@ -190,4 +195,7 @@ export class BatchSpanProcessor implements SpanProcessor {
       this._timer = undefined;
     }
   }
+
+  abstract onInit(config?: T): void;
```

Comment:
Why make these abstract if they're only used by browser? Wouldn't it make more sense to just override the constructor in the browser impl then call `onInit` after `super`?